website/content/blog/2024-04-10-meeting.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Board Meeting Minutes
+date: "2024-04-10"
+excerpt: Short update this month to try and speed up bounty allocation and check progress.
+tags: foundation
+author: MonoGame Foundation
+image: /images/blog/cover/monogame_foundation.png
+ogImage: /images/blog/cover/monogame_foundation.png
+---
+
+## Attendee's
+
+- Dean Ellis - President
+- Dominique Louis - Treasurer
+- Simon Jackson - Corporate Secretary
+- Tom Spilman - Banking Matters
+- Marko Jeremic - Board Chairman
+- Thomas Altenburger - Board Member
+
+## Agenda
+
+- Outstanding Bounty reviews
+- New / outstanding bounty requests
+- Metal implementation update
+
+## Outcomes
+
+A shorter meeting this month due to schedules and timings as the entire team are wrapped up with the new year business in their respective full-time jobs.  Still time however to go through the critical bounty requests to push the framework forward.
+
+- Bounty Updates:
+  - Only bounty creation will be discussed on the monthly meetings, including funding.
+  - An applicant will be allowed to pick up a bounty a minimum of one week after its creation (to allow for multiple applications).
+  - The president is responsible for getting the applicant to sign an NDA and tell them that they have been accepted for the task.
+  - The bounty creator is responsible for helping the applicant with what is requested from the bounty, selecting the candidate as well as ensuring the bounties work is completed properly.
+
+This will ensure a fair and speedy process and reduce any "red tape" for getting work picked up quickly, while still ensuring all interested applicants voices are heard.
+
+- Metal implementation is fine, so long as it follows the new native api.
+
+## Bounty tracker
+
+All bounties are still tracked primarily from the [Bounties](https://github.com/MonoGame/MonoGame/issues/8120) GitHub issue.  Each bounty has its own issue to keep communication open (subject to NDA requirements), including:
+
+- [A shared base for public and console repos](https://github.com/MonoGame/MonoGame/issues/8242) - **Still open**
+
+Upcoming bounties:
```

Comment:
If you are already mentioning the upcoming bounties, they are:
- MonoGame.Tool.X / MonoGame.Library.X for texture compression (waiting on ffmpeg bounty)
- Content Pipeline library hookup of all the Tool.X / Library.X repos (waiting on the above bounty)
- FAudio hookup (waiting on the native base bounty)